ts/features/itwallet/credentials/store/reducers/index.ts

```diff
@@ -41,6 +41,7 @@ const reducer = (
       }
 
       return {
+        ...state,
```

Comment:
Can we remove this unnecessary spread and restore the line before the `default` case , so this file won't result modified?